tests/unit/ReportUtilsTest.ts

```diff
@@ -339,6 +353,76 @@ describe('ReportUtils', () => {
             };
             expect(ReportUtils.requiresAttentionFromCurrentUser(report)).toBe(true);
         });
+
+        it('returns false if the user is not on free trial', async () => {
+            await Onyx.multiSet({
+                [ONYXKEYS.NVP_LAST_DAY_FREE_TRIAL]: null, // not on free trial
```

Comment:
null or undefined?

Reply:
When using set or merge we pass `null` when we want to clear that key.